tests/phpunit/payment-tokens/class-wc-stripe-payment-tokens-test.php

```
@@ -383,6 +383,56 @@ public function test_add_token_to_user_refreshes_cc_metadata_on_fingerprint_matc
+		$customer = new WC_Stripe_Customer( $user_id );
+
+		$reflection = new ReflectionMethod( WC_Stripe_Payment_Tokens::class, 'add_token_to_user' );
+		$reflection->setAccessible( true );
+		$result = $reflection->invoke( $this->stripe_payment_tokens, $incoming_pm, $customer, [] );
+
+		$this->assertSame( $seed_token_id, $result->get_id() );
+		$this->assertSame( 'apple_pay', $result->get_wallet_type(), 'wallet_type must be refreshed on duplicate-match.' );
+
+		$reloaded = WC_Payment_Tokens::get( $seed_token_id );
+		$this->assertSame( 'apple_pay', $reloaded->get_wallet_type(), 'wallet_type must be persisted to storage.' );
+	}
```

PHPMD (2.15.0)

[error] The method test_add_token_to_user_persists_wallet_type_on_refresh is not named in camelCase. (undefined)

(CamelCaseMethodName)